examples/myubuntu.id.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: myubuntuid
+  name: myubuntuid
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: myubuntuid
+  template:
+    metadata:
+      annotations:
+        admission.trusted.identity/inject: "true"
+        tsi.identities: |
+         - tsi.keycloak/server: "http://tsi-keycloak-v001.tsi-fra02-5240a919746a818fd9d58aa25c34ecfe-0000.eu-de.containers.appdomain.cloud"
```

Comment:
maybe lets call this `tsi.keycloak.token-url`? as per comment above changing to full token path.